apps/nextjs-website/src/components/molecules/BlocksRendererClient/BlocksRendererClient.tsx

```diff
@@ -56,7 +57,12 @@ const BlocksRendererClient = ({
           </Typography>
         ),
         heading: ({ children, level }) => (
-          <Typography marginY={4} variant={`h${level}`} color={textColor}>
+          <Typography
+            marginY={4}
+            variant={`h${level}`}
+            color={textColor}
+            id={computeId('blockRenderer', children)}
+          >
             {children}
           </Typography>
```

Comment:
with a div or with a Box

```suggestion
          <div
              id={computeId('blockRenderer', children)}
              style={{ marginTop: `-${scrollOffset}px`, paddingTop: `${scrollOffset}px` }}
          >
                    <Typography
                      marginY={4}
                      variant={`h${level}`}
                      color={textColor}
                    >
                      {children}
                    </Typography>
          </div>
```